gotpm/tests/seal_test.go

```diff
@@ -38,10 +38,14 @@ func TestSealPlain(t *testing.T) {
 
 	tests := []struct {
 		name   string
+		algo   string
 		pcrArg string
 	}{
-		{"Plain", ""},
-		{"PCRs", "7,8"},
+		{"Default", "", ""},
```

Comment:
I like this structure!

I don’t think we need 5 subtests here though.

RSA, RSAWithPCRs, and ECCWithPCRs is enough

Reply:
done